src/Tribe/Tickets_View.php

```diff
@@ -241,8 +290,10 @@ public function intercept_content( $content ) {
 	 * @return string           The correct File path for the tickets endpoint
 	 */
 	public function intercept_template( $old_file, $template ) {
+		global $wp_query;
+
 		// When it's not our query we don't care
-		if ( ! tribe_is_event_query() ) {
+		if ( empty( $wp_query->tribe_is_event_query ) ) {
```

Comment:
Same question about the switch away from [`tribe_is_event_query()`](https://github.com/moderntribe/the-events-calendar/blob/57c50037b6e91535b5108dcc9f1736a54d9679dd/src/functions/template-tags/query.php#L51) as asked above.